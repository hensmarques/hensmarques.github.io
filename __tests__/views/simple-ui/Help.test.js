import React from 'react';
import Help from '../../../src/views/simple-ui/Help';

describe('<Help/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Help />);
    expect(wrapper).toMatchSnapshot();
  });
});
