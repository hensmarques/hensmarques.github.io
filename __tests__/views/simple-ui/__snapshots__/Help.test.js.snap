// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Help/> renders and matches the snapshot 1`] = `
<Connect(SimpleUiWrapper)
  description="If the following does not provide you with the answer to your inquiry, don't hesitate to contact us, we will be glad to help you."
  pageClass="help-center"
  pageHeader="Frequently Asked Questions"
  title="Help Center"
  translation="HELP"
>
  <div
    id="0"
    key="0"
    name="#0"
  >
    <HelpAnswer
      question="How do I buy Bitcoin?"
    >
      <ol>
        <li>
          "Buy/Sell"
        </li>
        <li>
          Select the amount you wish to buy
        </li>
        <li>
          Choose the wallet you want to fund
        </li>
        <li>
          Choose your funding method
        </li>
        <li>
          Confirm your order and click "Buy"
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="1"
    key="1"
    name="#1"
  >
    <HelpAnswer
      question="How do I sell Bitcoin?"
    >
      <ol>
        <li>
          Go the "Buy/Sell"
        </li>
        <li>
          Select the "Sell Bitcoin" tab
        </li>
        <li>
          Choose the amount you want to sell
        </li>
        <li>
          Choose the wallet to sell from
        </li>
        <li>
          Choose the account to fund
        </li>
        <li>
          Confirm your order and click "Sell"
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="2"
    key="2"
    name="#2"
  >
    <HelpAnswer
      question="How do I send Bitcoin?"
    >
      <ol>
        <li>
          Click on "Send/Request"
        </li>
        <li>
          Enter the amount to send
        </li>
        <li>
          Enter the recipient's Bitcoin or email address
        </li>
        <li>
          Add a note
        </li>
        <li>
          Click "Send Bitcoin"
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="3"
    key="3"
    name="#3"
  >
    <HelpAnswer
      question="How do I Request Bitcoin?"
    >
      <ol>
        <li>
          Click on "Send/Request"
        </li>
        <li>
          Click on the "Request Bitcoin" tab
        </li>
        <li>
          Enter the amount to request
        </li>
        <li>
          Enter the sender's Bitcoin or email address
        </li>
        <li>
          Add a note
        </li>
        <li>
          Click "Request Bitcoin"
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="4"
    key="4"
    name="#4"
  >
    <HelpAnswer
      question="How do I deposit money into my account?"
    >
      <ol>
        <li>
          Click on "Deposit/Withdraw" link
        </li>
        <li>
          Click on the "Deposit" button for the account you wish to deposit funds to
        </li>
        <li>
          Fill up the short form
        </li>
        <li>
          Click Submit
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="5"
    key="5"
    name="#5"
  >
    <HelpAnswer
      question="How do I deposit money into my account?"
    >
      <ol>
        <li>
          Click on "Deposit/Withdraw" link
        </li>
        <li>
          Click on the "Deposit" button for the account you wish to deposit funds to
        </li>
        <li>
          Fill up the short form
        </li>
        <li>
          Click Submit
        </li>
      </ol>
    </HelpAnswer>
  </div>
  <div
    id="6"
    key="6"
    name="#6"
  >
    <HelpAnswer
      question="How do I deposit money into my account?"
    >
      <p>
        Bitcoin is a ground-breaking protocol, capable of changing how we
      buy and sell goods and even more, including the way we vote,
      identify ourselves, support causes, and have a huge impact on the
      world economy. We are at the bringe if incredible innovations.
      </p>
    </HelpAnswer>
  </div>
  <div
    id="7"
    key="7"
    name="#7"
  >
    <HelpAnswer
      question="Can I use Bitcoin anonymously?"
    >
      <p>
        It is not completely anonymous, but offers a great level of privacy
      together with transparency. It is not as anonymous as hard-cash, but
      its significantly more than credit card or bank transactions. Also,
      there are tools available and under development that promote privacy
      through Bitcoin, as well as technologies to protect against
      financial corruption and voting systems.
      </p>
    </HelpAnswer>
  </div>
  <div
    id="8"
    key="8"
    name="#8"
  >
    <HelpAnswer
      question="Can I walk into a store and buy using Bitcoin?"
    >
      <p>
        Yes. Real stores are starting to accept Bitcoin as a payment
      currency, and its number increases by the day. Many online services
      offer convenient applications for both the consumer and the store to
      buy and sell very easily using Bitcoin. Now you can find many ATMs
      selling Bitcoin, but the number of 2-way ATMs (the ones that sell
      and buy) is increasing.
      </p>
    </HelpAnswer>
  </div>
  <div
    id="9"
    key="9"
    name="#9"
  >
    <HelpAnswer
      question="What is Bitcoin Mining?"
    >
      <p>
        Similar to gold mining, Bitcoin mining is the process by which you
      digitally dig for Bitcoin, by running special scripts with
      algorithmic equations and hash functions that verify transactions on
      the blockchain. This process allows the confirmation of
      transactions, and adds new Bitcoin to the blockchain, and as a
      reward the miner receives Bitcoin.
      </p>
    </HelpAnswer>
  </div>
</Connect(SimpleUiWrapper)>
`;
